model/schema.json

```diff
@@ -52,6 +52,26 @@
         "name"
       ]
     },
+    "server-metadata": {
```

Comment:
Lets use `_` to be consistent with other fields in Beats / ECS.

Reply:
Why not pack this into server.metadata?

Reply:
This is the same as my comment above. This is just a reference.